media/gpu/sandbox/hardware_video_decoding_sandbox_hook_linux.cc

```diff
@@ -183,6 +183,10 @@ bool HardwareVideoDecodingPreSandboxHookForV4L2(
 //   (at least).
 bool HardwareVideoDecodingPreSandboxHook(
```

Comment:
Instead of modifying this file, could we just not call this (and the sibling for encoding) from https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:content/utility/utility_main.cc;l=229-238;drc=ac02e200288bef9c817fad9b06763463e4e9493d ? 

I think this should be gated upstream with a flag but it isn't and might be complicated. 

Ideally we'd remove the whole media/gpu dependencies from all over the place, but that also needs touches in content etc.